velox/exec/RowContainer.h

```diff
@@ -316,15 +336,26 @@ class RowContainer {
              : 0);
   }
 
+  /// Removes the data of a given row from the column stats by iterating
+  /// over each column in the row and updates the column stats.
+  ///
+  /// @param row - The row from which the column stats are to be updated.
+  /// @param keepNullCount - If true, the null count is kept if originally null
+  /// or increase if originally not null.
+  void removeRowFromColumnStats(const char* row, bool keepNullCount);
```

Comment:
Shall we separate the 2 functionalities? When keepNullCount is true we are technically not removing a row. Taking the callsite as an example, when setting an entire row to null, it basically removes the entire row and adds a null row.

Reply:
Should we make this a private method?

Reply:
From the usage, it is to replace with a null row. I named it to keepNullCount to be more general so it provides a mechanism instead of a use case. But as you said, there is currently no other use cases and it is actually indeed only private method. So I think it makes sense to change it to replaceWithNullRow.